### ISSUE TITLE
Existing Ingress api version has been deprecated

### ISSUE THREAD

**Issue by karezza** [Mailu/helm-charts#107]
**Describe the bug**
With kubernetes v1.22.* 'networking.k8s.io/v1beta1' is no longer available causing the helm install to fail.

**Environment**
```
$ kubectl version 
Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.1", GitCommit:"632ed300f2c34f6d6d15ca4cef3d3c7073412212", GitTreeState:"clean", BuildDate:"2021-08-19T15:45:37Z", GoVersion:"go1.16.7", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.1", GitCommit:"632ed300f2c34f6d6d15ca4cef3d3c7073412212", GitTreeState:"clean", BuildDate:"2021-08-19T15:39:34Z", GoVersion:"go1.16.7", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
```
$ ./update-install.sh 
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "mailu" chart repository
...Successfully got an update from the "bitnami" chart repository
Update Complete. ⎈Happy Helming!⎈
Release "mailu" does not exist. Installing it now.
Error: unable to build kubernetes objects from release manifest: [unable to recognize "": no matches for kind "Ingress" in version "networking.k8s.io/v1beta1", unable to recognize "": no matches for kind "Certificate" in version "cert-manager.io/v1alpha2"]
```

**Comment by karezza** [comment]
Able to work around this by creating my own ingress and adding to the values.yaml:
```
ingress:
  externalIngress: false
```

**Comment by karezza** [comment]
Modified the ingress template to work with kubernetes 1.22.1, not sure if there is a goal to try and support older apiVersions along with the new one.  In any case, here's what worked for me.  Probably want a variable for the ingressClassName:

```
$ more helm-charts/mailu/templates/ingress.yaml 
# This file is derived from https://github.com/Mailu/Mailu/blob/master/docs/kubernetes/mailu/admin-ingress.yaml

{{- $fullname := (include "mailu.fullname" .) }}
{{ if .Values.ingress.externalIngress }}
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: {{ $fullname }}-ingress
{{- if .Values.ingress.annotations }}
  annotations:
{{ toYaml .Values.ingress.annotations | indent 4 }}
{{- end }}
  labels:
    app: {{ $fullname }}
    component: admin
spec:
  ingressClassName: nginx
  tls:
  - secretName: {{ $fullname }}-certificates
    hosts:
{{- range .Values.hostnames }}
    - "{{ . }}"
{{- end }}
  rules:
{{- range .Values.hostnames }}
  - host: "{{ . }}"
    http:
      paths:
      - path: "/"
        pathType: Prefix
        backend:
          service:
            name: {{ $fullname }}-front
            port:
              number: 80
{{- end }}
{{ end }}
```

**Comment by micw** [comment]
Fixed in v1.3
